packages/wc-provider/block-tracker.ts

```diff
@@ -12,6 +12,23 @@ export default class PatchedSubscribeBlockTracker extends SubscribeBlockTracker
     //@ts-ignore need to expose the provider for updating
     return this._provider;
   }
+
+  // Some RPC nodes don’t support subscribing to newHeads with {} parameters, so this removes that
+  protected async _start(): Promise<void> {
+    //@ts-ignore access private this._subscriptionId
+    if (this._subscriptionId === undefined || this._subscriptionId === null) {
```

Comment:
a more succinct way to express this is:
`this._subscriptionId == null`, its the one time that `==` makes sense to use.